NOTICE

```diff
@@ -163,6 +164,7 @@ This project includes:
   Jetty :: XML utilities under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
   Jetty Server under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
   Jetty Utilities under Apache Software License - Version 2.0 or Eclipse Public License - Version 1.0
+  JLine under BSD
   JSch under BSD
   json4s-ast under ASL
   json4s-core under ASL
```

Comment:
I had to generate a new notice file,
this line got removed:
-  jdk.tools:jdk.tools under Oracle Binary Code License Agreement for the Java SE